### PR TITLE
solana end to end

### PR DESCRIPTION
**Pull request by enriquesouza**

### CODE REVIEW

File: frontend/src/model/chain.ts [hunk 1]
```diff
@@ -45,10 +45,12 @@ export class ChainUtils {
     }
     return undefined;
   }
+
   static getTssKeysignType(chain: Chain): TssKeysignType {
     switch (chain) {
-      case (Chain.Solana, Chain.Polkadot):
+      case "Solana":
```
Comment (johnnyluo) [inline frontend/src/model/chain.ts:51]: why change it to "Solana" , not use `Chain.Solana`?

Comment (enriquesouza) [reply frontend/src/model/chain.ts:51]: It was always returning the default;

Comment (enriquesouza) [reply frontend/src/model/chain.ts]: Changed to use the enum.

**Review by johnnyluo — APPROVED** [review]
LGTM

**Pull request merged**